### DISCUSSION TITLE
mini版好像无法进入webui，laster版正常？

### DISCUSSION THREAD

**Discussion by 518512** [haiibo/OpenWrt#153] [General]
请问是什么原因?

**Comment by 518512** [comment]
能进去一次，重启容器以后就进不去了

**Comment by 518512** [comment]
uhttpd 日志显示  Failed to initialize ustream-ssl

![uhttp log](https://github.com/ophub/amlogic-s9xxx-openwrt/assets/7768188/2f515d78-82a4-4269-8eda-d3d8ad1f50d3)

**Comment by Java2Boy** [comment]
遇到同样问题，MINI版修改好IP后无法进入luci界面，latest没问题

**Reply by 518512** [reply]
现在的临时解决办法就是， 编辑 /etc/config/uhttpd  ，注销掉两个 https 相关选项。

近期很多镜像都有这个问题
